packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -4,218 +4,231 @@ title: OAuth 2.0 and OpenID Connect Overview
 
-Although OpenID Connect is built on top of OAuth 2.0, the [OpenID Connect specification](https://openid.net/connect/) uses slightly different terms for the roles in the flows:
+Although OIDC extends OAuth 2.0, the [OIDC specification](https://openid.net/connect/) uses slightly different terms for the roles in the flows:
 
 * **OpenID provider**: The authorization server that issues the ID token. In this case Okta is the OpenID provider.
 * **end user**: The end user's information that is contained in the ID token.
 * **relying party**: The client application that requests the ID token from Okta.
-* **ID token**: The token issued by the OpenID Provider and contains information about the end user in the form of claims.
+* **ID token**: The token issued by the OpenID provider and contains information about the end user in the form of claims.
 * **claim**: The claim is a piece of information about the end user.
 
 The high-level flow looks the same for both OpenID Connect and regular OAuth 2.0 flows. The primary difference is that an OpenID Connect flow results in an ID token, in addition to any access or refresh tokens.
 
-## Choosing an OAuth 2.0 flow
+## Choose an OAuth 2.0 flow
+
+The OAuth flow that you use depends on your use case. The following sections recommend OAuth 2.0 flows based on:
 
-Which OAuth flow that you use depends on your use case. The table below maps application types to our recommended OAuth 2.0 flows. If you'd like more information, keep reading for help with choosing an OAuth flow based on (1) the type of token that you need, and/or (2) the type of client application that you are building.
+* The type of application that you're building and the token types that the application requires
+* The type of client that you're building
 
-### Recommended flow by application type
+### What type of application are you building?
 
-The table shows you which OAuth 2.0 flow to use for the type of application that you are building.
+The following table shows you which OAuth 2.0 flow to use for the type of application that you’re building.
 
-| Type of Application     | OAuth 2.0 flow / grant type                      | Access Token?    | ID Token?     |
+| Type of application     | OAuth 2.0 flow / grant type                      | Access token?    | ID token?     |
 | ----------------------- | ------------------------------------------------ | ---------------- | ------------- |
-| Server-side (aka Web), <br><br> Single-Page Application, <br><br> or Native | [Authorization Code with PKCE](#authorization-code-flow-with-pkce) or [Interaction Code](#interaction-code-flow) (Identity Engine only).                | &#9989;          | &#9989;      |
+| Server-side (aka web), <br />Single-Page Application, <br />or Platform-specific | [Authorization Code with PKCE](#authorization-code-flow-with-pkce) or<br> [Interaction Code](#interaction-code-flow) (Identity Engine only).                | &#9989;          | &#9989;      |
```

Comment:
Where did "platform-specific" come from? That isn't a term we usually use in OAuth.

Reply:
@aaronpk We're actively trying to avoid using the word "Native" for inclusivity reasons on the docs. "Platform-specific" is the standard alternative adjective as it covers mobile and desktop apps.

Reply:
Agreed upon Mobile or Desktop instead.